actionpack/lib/action_dispatch/system_test_case.rb

```diff
@@ -2,6 +2,7 @@
 
 gem "capybara", ">= 3.26"
 
+require "capybara/rails"
```

Comment:
We only want the rack_test driver, so it is better to define it on https://github.com/rails/rails/blob/c91adcd3f3414890713da4ff1d0f61b70fd59447/actionpack/lib/action_dispatch/system_testing/driver.rb